lacework/resource_lacework_integration_gcp_agentless_scanning.go

```diff
@@ -265,6 +273,8 @@ func resourceLaceworkIntegrationGcpAgentlessScanningCreate(d *schema.ResourceDat
 		d.Set("created_or_updated_by", integration.CreatedOrUpdatedBy)
 		d.Set("type_name", integration.Type)
 		d.Set("org_level", integration.IsOrg == 1)
+		d.Set("server_token", integration.ServerToken)
+		d.Set("uri", integration.Uri)
```

Comment:
there two `Set()`s need to be added to `resourceLaceworkIntegrationGcpAgentlessScanningRead()` and probably also to `resourceLaceworkIntegrationGcpAgentlessScanningUpdate()` - just to be safe.

Reply:
Talking with @ammarekbote over Slack, he mentioned that the token won't be returned on GET and PATCH requests for security reasons, therefore adding the server token only in POST requests (create) is the right thing to do, having said that, the URI is shown all the time so let's just add URI on Read/Update functions here.

Reply:
I see that we populate both these fields for `update` too - so setting both there and for `read` just setting Uri